crates/python/qcs_sdk/qpu/translation.pyi

```diff
@@ -55,26 +32,26 @@ def get_quilt_calibrations(
     quantum_processor_id: str,
     client: Optional[QCSClient] = None,
     timeout: Optional[float] = None,
-) -> QuiltCalibrations:
+) -> str:
```

Comment:
To avoid the breaking change, couldn't we re-parse the program?

Reply:
I guess the better question is whether we want to avoid the breaking change or not.

Reply:
`QuiltCalibrations` was derived from an OpenAPI type (and contained the program string, not a parsed Program), so I think a break is non-negotiable, since it would be weird to construct a to-be-deprecated OpenAPI struct.